### DISCUSSION TITLE
An unequal fight against the program :(

### DISCUSSION THREAD

**Discussion by eBazyl** [PrestaShop/PrestaShop#34451] [Q&A]
After migrating from 16.1.24 to 8.1.1, I had several errors: no categories in BO, zero prices, incorrectly assigned ROOT, ....
I managed to restore the store to a relatively working order, but just because it works doesn't mean it's error-free.

"Unable to generate a URL for the named route "admin_logos_index" as such route does not exist." appeared in BO when I browsed categories or products:
![2023-11-02 (1)](https://github.com/PrestaShop/PrestaShop/assets/148250863/2fd1f8cd-bc78-4991-80f4-34855a205295)
but I could live with it.
But after two days I got an error - I couldn't enter the orders:
"Several errors "User Notice: escape: unsupported type: UTF-8 - returning unmodified stringin" in debug mode",
...i fixed it by:
\vendor\smarty\smarty\libs\plugins\modifier.escape.php - // trigger_error("escape: unsupported type: $esc_type - returning unmodified string", E_USER_NOTICE);
... and an error popped up:
"An exception has been thrown during the rendering of a template ("Warning: Undefined array key "weight_adr"")."

I've been working on the program for almost a month now and I'm really fed up with it, so I came up with the idea of upgrading to 8.1.2 with 1 click.
I deleted all the modules I added and when I updated with the option "Disable non-built-in modules NO", these bugs moved to version 8.1.2, but when I updated with the option: ""Disable non-built-in modules YES", then I have a message:
"The AdminPsEditionBasicHomepageController controller is unavailable or damaged.", but I have access to orders :)

I've been browsing the forums for a month and I see that a lot of people have post-immigration problems. I've seen various little tips that solve small problems, but hasn't anyone put it all together into a fix. Or maybe there are some tools to check the correctness of the database structure for 8.xx?
I'm asking for advice or some advice, because when you solve one problem, another one appears and so on endlessly :(

**Comment by kpodemski** [comment]
Hello @eBazyl 

It sounds like you took quite a big project on your shoulders! Have you considered hiring an experienced developer to help you with those issues? Migrating from 1.6 to 8.1 might be or I'd rather say **is** challenging project.
